sdks/python/apache_beam/io/avroio_test.py

```diff
@@ -281,30 +277,34 @@ def test_read_with_splitting_multiple_blocks(self):
 
   def test_split_points(self):
     num_records = 12000
-    file_name = self._write_data(count=num_records)
+    if self.use_fastavro:
+      sync_interval = 16000
```

Comment:
Let's not pass sync_interval here, so that the default value is used, see also a comment below on this.

Reply:
Discussed offline - the test code relies on sync_interval.